src/components/Listing/Listing.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch} from "react-redux";
+
+import Select from "react-dropdown-select";
+import ReactPaginate from "react-paginate";
+
+import { fetchData } from "../../store/api-call";
+import ShowList from "../ShowList/ShowList";
+
+import classes from "./Listing.module.css";
+import { options } from "../../assets/apis/config";
+
+import { pageRangeDisplayed } from "../../assets/apis/config";
+import { RiseLoader } from "react-spinners";
+
+const Listing = (props) => {
```

Comment:
It is better to not use ing form for file name, isn't `PagedList` a better name?

Reply:
done